helm/minio/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Multi-Cloud Object Storage
 name: minio
-version: 5.0.4
+version: 5.1.0
```

Comment:
do not update version yourself, this is managed by us @sathieu please revert.

Reply:
@harshavardhana Done